cli/internal/logstreamer/logstreamer.go

```diff
@@ -125,9 +125,33 @@ func (l *Logstreamer) out(str string) {
 		str = l.colorOkay + l.prefix + l.colorReset + " " + str
 	} else if l.prefix == "stderr" {
 		str = l.colorFail + l.prefix + l.colorReset + " " + str
-	} else {
-		str = l.prefix + str
 	}
 
 	l.Logger.Print(str)
 }
+
+// PrettyStdoutWriter wraps an ioWriter so it can add string
+// prefixes to every message it writes to stdout.
+type PrettyStdoutWriter struct {
+	w      io.Writer
+	Prefix string
+}
+
```

Comment:
[Verify interface implementation](https://github.com/uber-go/guide/blob/master/style.md#verify-interface-compliance)
```suggestion

var _ io.Writer = (*PrettyStdoutWriter)(nil)

```